pkg/lease/manager_test.go

```diff
@@ -72,7 +79,17 @@ var _ = Describe("Leases", func() {
 			_, err := manager.GetLease(context.TODO(), node)
 			Expect(err).NotTo(HaveOccurred())
 
-			err = manager.RequestLease(context.Background(), node, leaseDuration)
+			switch actionOnLease {
+			case requestLease:
+				{
```

Comment:
unneeded {} in both case blocks

Reply:
Hmm I don't follow, both `requestLease` and `invalidateLease` has { }.
Am I missing something ?